test/unit/groupTest.js

```diff
@@ -0,0 +1,116 @@
+const helpers = require ('../helpers/createDanceAPI');
+const test = require('tape');
+
+test('changing dance moves for all updates all dancers', async t => {
+  const nativeAPI = await helpers.createDanceAPI();
+  nativeAPI.play({
+    bpm: 120,
+  });
+  nativeAPI.setAnimationSpriteSheet("CAT", 0, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("CAT", 1, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("BEAR", 0, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("BEAR", 1, {}, ()=> {} );
```

Comment:
Tiny style nit: 
```suggestion
  nativeAPI.setAnimationSpriteSheet("BEAR", 1, {}, () => {});
```

(A few other places in this PR as well.)